src/components/Form.js

```diff
@@ -172,8 +172,19 @@ class Form extends React.Component {
     validate(values) {
         FormActions.setErrors(this.props.formID, null);
         FormActions.setErrorFields(this.props.formID, null);
+
+        // Run any validations passed as a prop
         const validationErrors = this.props.validate(values);
 
+        // Validate the input for html tags. It should superseed any other error
+        _.every(values, (inputValue, inputID) => {
+            if (!inputValue || inputValue.search(/<(.|\n)*?>/g) === -1) {
```

Comment:
The `CONST.js` file has `VALIDATE_CODE_REGEX_STRING`. Could you please move this regex to the same place and define it there?